libopenage/curve/tests/curve_types.cpp

```diff
@@ -386,6 +387,86 @@ void curve_types() {
 		TESTEQUALS(c.get(1), 0);
 		TESTEQUALS(c.get(5), 0);
 	}
+
+	{ // array
+		auto f = std::make_shared<event::EventLoop>();
+
+		Array<int, 4> a(f,0);
+		a.set_insert(time::time_t(1), 0, 0);
```

Comment:
The explicit construction of `time::time_t` is unnecessary for calling the functions.

```suggestion
		a.set_insert(1, 0, 0);
```